beta-src/src/components/ui/WDCountryTable.tsx

```diff
@@ -0,0 +1,177 @@
+  {
+    id: "centerQty",
+    label: "Centers",
+    icon: CentersIcon,
+    align: "center",
+  },
+  {
+    id: "bet",
+    label: "Bet",
+    icon: BetIcon,
+    align: "center",
+  },
+  {
+    id: "delaysLeft",
+    label: "Delays",
+    icon: DelaysIcon,
+    align: "center",
+  },
+];
+
+const WDCountryTable: React.FC<WDCountryTableProps> = function ({
+  countries,
+  device,
+  maxDelays,
+}): React.ReactElement {
+  const mobileLandscapeLayout =
+    device === Device.MOBILE_LANDSCAPE || device === Device.MOBILE_LG_LANDSCAPE;
+  const WDTableCell = styled(TableCell)(() => {
+    const padding = mobileLandscapeLayout ? "6px 6px" : "6px 16px";
+    return {
+      [`&.${tableCellClasses.head}`]: {
+        lineHeight: "unset",
+        borderBottom: 0,
+        padding,
+      },
+      [`&.${tableCellClasses.body}`]: {
+        borderBottom: 0,
+        padding,
+      },
+    };
+  });
+  return (
+    <div>
+      <Table stickyHeader size="small" aria-label="country info table">
+        <TableHead sx={{ height: "70px" }}>
+          <TableRow sx={{ verticalAlign: "top" }}>
+            {columns.map((column) => (
+              <WDTableCell key={column.id} align={column.align}>
+                {column.icon && <column.icon />}
+                <Box sx={{ fontSize: 10, fontWeight: 400 }}>
+                  {column.label.toUpperCase()}
+                </Box>
+              </WDTableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {countries.map((country) => (
+            <React.Fragment key={country.power}>
+              <TableRow>
+                {columns.map((column) => {
+                  const style = {
+                    color: "black",
+                    fontWeight: 400,
+                  };
+                  let value: string;
+                  switch (column.id) {
+                    case "power":
+                      value = mobileLandscapeLayout
+                        ? country.abbr.toUpperCase()
+                        : country.power.toUpperCase();
+                      style.color = country.color;
+                      style.fontWeight = 700;
+                      break;
+                    case "delaysLeft":
+                      value = `${country[column.id]}/${maxDelays}`;
+                      break;
+                    default:
+                      value = `${country[column.id]}`;
+                      break;
+                  }
+                  return (
+                    <WDTableCell key={column.id} align={column.align}>
+                      <span style={style}>{value}</span>
+                    </WDTableCell>
+                  );
+                })}
+              </TableRow>
+              {country.votes && (
+                <TableRow>
+                  <WDTableCell
+                    sx={{
+                      fontSize: "70%",
+                      paddingTop: "0px !important",
```

Comment:
why?

Reply:
Nothing else was working, open to suggestions.

Reply:
hmm, that is weird. ok. can you try setting padding instead of padding top? i just hate !important cause it means we aren't cascading right for some reason. but, if this is all that works, then...

Reply:
If I set padding then I'll be overwriting the padding set by mobileLandscapeLayout at the top of the component.